lib/Tool/Newsletter.php

```diff
@@ -137,8 +143,6 @@ public static function sendNewsletterDocumentBasedMail(Mail $mail, SendingParamC
 
         if (!empty($mailAddress)) {
             $mail->setTo($mailAddress);
-            // Getting bounces
```

Comment:
Not necessary, as the SMTP Server will set it automatically set the return path when finally sending the email.

https://www.postmastery.com/about-the-return-path-header/

Reply:
You added this in #4349 ... can't hurt to leave it as it is, right?

Reply:
I know it was me. :-) 
But on what I read in the meantime, I would recommend to leave it out, because some mailserver seems to reject or identify the email as spam, when return path is wrongly set. At the end if we leave it in, it won't hurt the sending process, !maybe! some emails won't receive the customer.